sdks/python/apache_beam/testing/benchmarks/nexmark/nexmark_launcher.py

```diff
@@ -170,6 +182,32 @@ def parse_args(self):
         choices=['PUBLISH_ONLY', 'SUBSCRIBE_ONLY', 'COMBINED'],
         help='Pubsub mode used in the pipeline.')
 
+    parser.add_argument(
+        '--exportSummaryToInfluxDB',
+        default=False,
+        action='store_true',
+        help='If set store results in influxdb')
+    parser.add_argument(
+        '--influxDatabase',
+        type=str,
+        default='beam_test_metrics',
+        help='Influx database name')
+    parser.add_argument(
+        '--influxHost',
+        type=str,
+        default='http://localhost:8086',
+        help='Influx database url')
+    parser.add_argument(
+        '--baseInfluxMeasurement',
+        type=str,
+        default='nexmark',
+        help='Prefix to influx measurement')
+    parser.add_argument(
+        '--influxRetentionPolicy',
+        type=str,
+        default='forever',
+        help='Retention policy for stored results')
```

Comment:
nit: can you use snake case for consistency with the other args? Or do we need to use camel case for compatiblity with the java version

Reply:
I made it camel case so I wouldn't have to translate the Java args, but I can do that instead.